firedrake/parameters.py

```diff
@@ -39,6 +56,17 @@ def rename(self, name):
                           simd_isa=os.environ.get('PYOP2_SIMD_ISA', 'sse'),
                           O2=True))
 
+pyop2_opts = Parameters("pyop2_options",
+                        debug=False,
+                        type_check=False,
+                        backend="sequential",
```

Comment:
No, that's the "shared memory parallelisation backend".  However, I shall remove this explicit setting.